src/sentry/relocation/tasks/process.py

```diff
@@ -160,6 +163,12 @@
     retry_backoff=RETRY_BACKOFF,
     retry_backoff_jitter=True,
     soft_time_limit=FAST_TIME_LIMIT,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of `FAST_TIME_LIMIT` to align with the celery configuration?